rasa/core/policies/policy.py

```diff
@@ -195,6 +201,70 @@ def featurize_for_training(
 
         return state_features, label_ids, entity_tags
 
+    def prediction_states(
+        self,
+        tracker: DialogueStateTracker,
+        domain: Domain,
+        use_text_for_last_user_input: bool = False,
+        **kwargs: Any,
```

Comment:
Our out-of-the-box policies should unwrap default `kwargs` and make them explicit as soon as possible (which would be policy.predict_action_probabilities. This makes the code more explicit and also enables type checking